### DISCUSSION TITLE
Removing icon space in global menu

### DISCUSSION THREAD

**Discussion by terr72** [black7375/Firefox-UI-Fix#530] [Q&A]
Can someone give me a hint on how to remove the icons and their space in the global menus?
See the green rectangle in the screenshot below.

I managed to remove the icons by simply editing out the `** Global Menu` section, but the space for the icons remains.
How can I fix this?
Also: if there's a more elegant solution to disable the icons in the global menu, feel free to let me know. :)

![menu_remove_space](https://user-images.githubusercontent.com/23163919/207463067-fc405873-4983-4c8e-8ce5-085046555eec.png)

Edit: Just realized that setting entries like
```
userChrome.padding.global_menubar
userChrome.icon.global_menubar
```
in user.js to false do what I was looking for. So this is solved.

**Comment by black7375** [comment]
Yes, I've organized what users can ask often in wiki.
https://github.com/black7375/Firefox-UI-Fix/wiki/Tips#remove-menubar-icon
